### ISSUE TITLE
Error parsing modification string

### ISSUE THREAD

**Issue by wsnoble** [crux-toolkit/crux-toolkit#537]
The newest version of Comet produces peptides that can't be parsed by assign-confidence. The error message shown below repeats many many times, though the command runs to completion.
[Uploading comet.target.txt…]()

```
NFO: CPU: pyrrolysine.gs.washington.edu
INFO: Crux version: 4.1-69e42281-2021-12-01
INFO: Tue Dec  7 18:20:34 PST 2021
COMMAND LINE: ../../src/crux assign-confidence --output-dir comet --score e-value comet/comet.target.txt
INFO: Beginning assign-confidence.
INFO: Estimation method = tdc.
ERROR: Error parsing modification string: Could not parse modification string '-'
ERROR: Error parsing modification string: Could not parse modification string '-'
```

**Comment by kfattila** [comment]
I cannot get the comet.target.txt. Could you upload it again!

**Comment by wsnoble** [comment]
[comet.target.txt](https://github.com/crux-toolkit/crux-toolkit/files/7677846/comet.target.txt)

**Comment by wsnoble** [comment]
Resolved with commit 7586f912b25f224324606c50f210741af8bcb5f1
